database/migrations/2016_04_05_202051_add_app_infos_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddAppInfosTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('app_infos', function (Blueprint $table) {
```

Comment:
**Code Quality - Critical**
@andela-araimi This Migration is incomplete, there is no reference to the users table (this is the only way for any relationship on the tables (users and app_info) to work) and also please change the table name from `app_infos` to `app_info`. Thank you.